### DISCUSSION TITLE
1.41.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#24570] [General]
Feel free to discuss the [1.41.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.41.0) release here!

**Comment by pannal** [comment]
I don't see _any_ devices in the frontend after the upgrade. No errors in the logs. Downgrade back to 1.40.2 works well.

**Reply by pannal** [reply]
The frontend seems to fail with `index-74e84a13.js:61 WebSocket connection to 'wss://xxxxxxx/zigbee/api' failed: `

**Reply by pannal** [reply]
Issue found. This is due to the new `base_url` setting. Z2M is behind a reverse proxy and I had this config before:
```
    location ^~ /zigbee/api {
        proxy_pass         http://ip:port/api;
        proxy_set_header Host $host;

        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
    }

    location ^~ /zigbee {
        include snippets/proxy.conf;
        proxy_pass http://ip:port/;
    }
```

Now we have to set `base_url` to `/zigbee` in this case and change the NGINX config to:
```
    location ^~ /zigbee/api {
        proxy_pass         http://ip:port;
        proxy_set_header Host $host;

        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
    }

    location ^~ /zigbee {
        include snippets/proxy.conf;
        proxy_pass http://ip:port;
    }
```

**Comment by KirikauKiwi** [comment]
Not able to update the Ikea Inspiring smart plugs and existing problem with Aqara [DJT11LM](https://www.zigbee2mqtt.io/devices/DJT11LM.html#aqara-djt11lm) tilt sensor not showing readings since V1.4.0 or V1.4.1

**Reply by ClasheGit** [reply]
Hello, for my case, in 1.40.1-2 the axis of my aqara DJT11LM sensors no longer goes up. So I went back to 1.40.1-1, it worked again. Now, November 1st, update 1.41.0, all of my aqara sensors work perfectly. So I did the update from 1.40.1-1 directly to 1.41.0.

**Reply by pannal** [reply]
DJT11LM works 100% in 1.41.0 for me. All readings exist and update.

**Reply by KirikauKiwi** [reply]
Thanks, I took a further look at mine and it had disconnected from the Zigbee network, so all good 👍

**Comment by PlayFaster** [comment]
Hi. The new throttle option does not appear in the UI, but I can set it, and it works, in the YAML config. Is this (not in UI yet) expected behavior? 

Also, is there any intent to introduce a throttle_ignore option, similar to debounce_ignore? I think this would be very useful, on for example mmWave presence/occupancy sensors, where I would like to throttle the output from the sensor, but always (throttle_ignore) immediately publish changes to the presence attribute itself.

**Reply by mjasiak** [reply]
Hej. 
Anything new regarding that topic? I just fell into the same use case and would love to see that option implemented. Checked the google very quick, but could not find any other topic mentioning it.

**Comment by RangoDzh** [comment]
I found issue:

Device type
    EndDevice
Zigbee Model
    SNZB-03
Zigbee Manufacturer
    eWeLink
Description
    Motion sensor
Support status

    Supported
IEEE Address
    0xa4c138662d3deda1
Network address
    0xEC92 / 60562
Firmware version
    0122052017
Manufacturer
    [SONOFF](https://www.zigbee2mqtt.io/supported-devices/#v=SONOFF)
Model
    [SNZB-03](https://www.zigbee2mqtt.io/devices/SNZB-03.html#sonoff-snzb-03)


After today update, this device change type to water leak sensor, and stop working, after restore backup and repair, work normal with older version of package 1.40.2-1. Please solve.

**Comment by purche42** [comment]
Same problem with lights on/off which occurs first in 1.40.2 
see lights problem [#24293](https://github.com/Koenkk/zigbee2mqtt/issues/24293)

**Comment by chris-1243** [comment]
Hi,

I randomly see polling issue on SNZB-02P/D since the update. I have 5 of them working fine and they were not throwing any errors at all since the last 2  updates (4 02P and 1 02D). It started on 1.40.2 quite randomly and only soms errors in the entire month of october.
1.41.1 is more or less one error every 24h and mainly the same sensor. The values are reported correctly just the errors in the logs like:

```
[2024-11-04 17:59:22] error: 	zh:controller:device: Handling of poll check-in from 0x0cae5ffffeb05d2e failed (ZCL command 0x0cae5ffffeb05d2e/1 genPollCtrl.checkinRsp({"startFastPolling":false,"fastPollTimeout":0}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"reservedBits":0,"writeUndiv":false,"sendPolicy":"immediate"}) failed (Timeout - 59967 - 1 - 234 - 32 - 11 after 10000ms))
```

It is similar to #23166 but not the same devices. As a workaround, I might unbind the `genPollCtrl` cluster. No change in my setup as well...

**Comment by JJANOUSEK1** [comment]
Hi all, after update I have problem to display logs....I see only error 
![image](https://github.com/user-attachments/assets/b30ab27b-a1dc-45a9-ab02-77564c6a0cc3)
how to fix it? tks

**Comment by schwanenberg1969** [comment]
I have two instances running. Cannot delete either one. 

Tried stopping both, deleting image --> error: still running

Reboot the whole snyology. Same problem.

Anybody with a solution?

![kkoenkk1](https://github.com/user-attachments/assets/557ebcc7-654b-475f-91b6-3c0550b1821e)
![kkoenkk2](https://github.com/user-attachments/assets/ae1aab0b-15dd-4a7e-b547-06becb374aa1)

**Comment by RobLeighton67** [comment]
I can set the throttle option via MQTT and see it in the configuration.yaml but even after restarting Z2M the messages still pour in every second from my Fluid level sensor. The setting appears to be overridden by my global device_options: debounce 0.5 setting.
I have removed the global device_options debounce and the throttle works as expected.
I'd prefer my global option to be overridden by the local setting (debounce or throttle)?

**Comment by drogago** [comment]
Hello. When is the next release expected?
I have a device that is currently unsupported. However, after reviewing the source code, I noticed that support appears to have been added. I’m considering whether it would make sense to implement temporary support on my own.

I apologize for posting this in this topic, but I felt it wasn't significant enough to warrant creating a new topic.

**Reply by Koenkk** [reply]
Every 1st of the month
